src/client/app/components/AdminComponent.tsx

```diff
@@ -184,16 +183,7 @@ export default class AdminComponent extends React.Component<AdminProps, {}> {
 			showErrorNotification('Please select a meter');
 		} else {
 			const file = files[0];
-			const data = new FormData();
-			data.append('csvFile', file);
-			axios({
-				method: 'post',
-				url: `/api/fileProcessing/${this.props.selectedImportMeter.value}`,
-				data,
-				params: {
-					token: getToken()
-				}
-			})
+			metersApi.submitNewReadings(this.props.selectedImportMeter.value, file)
```

Comment:
Is there a reason directly below this line that you used a `.then` instead of `async`/`await`? Or would that get ugly fast because `await` needs to be in an async function?

Reply:
It's cleaner than wrapping it in an async IIFE in this case, and it was already like that, so I decided not to change it.